### PR TITLE
feature: attach-t0-prefix list, detach-t0-prefix-list

### PR DESCRIPTION
**Pull request by fr0gger03**
**New features:** 
- _attach-t0-prefix-list_: this will attach a specified prefix list to either the in_route_filter or the out_route_filter for the specified T0 BGP neighbor
- _detach-t0-prefix-lists_: this will detach all prefix lists from the the specified T0 BGP neighbor

tested on existing test SDDC

Signed-off-by: Tom Twyman <ttwyman@vmware.com>

### CODE REVIEW

File: pyVMC.py [hunk 1]
```diff
@@ -1607,6 +1607,57 @@ def newBGPprefixlist(csp_url, session_token):
...
+    if response.status_code == 200:
+        json_response = response.json()
+        neighbor_json = json_response
+        del neighbor_json['_create_time']
+        del neighbor_json['_create_user']
+        del neighbor_json['_last_modified_time']
+        del neighbor_json['_last_modified_user']
+        del neighbor_json['_system_owned']
+        del neighbor_json['_protection']
+        del neighbor_json['_revision']
```
Comment (kremerpatrick) [inline pyVMC.py:1624]: When hidden properties are skipped in this manner, it introduces future bugs when an additional hidden property gets added. Consider looping with a for loop and deleting properties that start with '_'

File: pyVMC.py [hunk 2]
```diff
@@ -1607,6 +1607,57 @@ def newBGPprefixlist(csp_url, session_token):
...
```
Comment (kremerpatrick) [inline pyVMC.py:1649]: When hidden properties are skipped in this manner, it introduces future bugs when an additional hidden property gets added. Consider looping with a for loop and deleting properties that start with '_'

**Review by kremerpatrick — CHANGES_REQUESTED** [review]
Nice additions! Little rework for the system property skipping but other than that looks good.

**Comment by fr0gger03** [comment]
Thanks - should be fixed. Please check it

**Review by kremerpatrick — APPROVED** [review]
LGTM

**Pull request merged**